instrumentation/net/http/otelhttp/transport.go

```diff
@@ -16,34 +16,45 @@ package otelhttp // import "go.opentelemetry.io/contrib/instrumentation/net/http
 
 import (
 	"context"
+	"go.opentelemetry.io/otel/metric"
```

Comment:
I wonder if this project has a formatting check in CI...

Reply:
It might be, I did most of this work on a plane with a 3 kb/s connection so unfortunately I wasn't able to download `golangci-lint` and reproduce the issues locally, I'll try and sort it out